src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -323,6 +365,101 @@ export class QuickBar extends LitElement {
     );
   }
 
+  private _generateNavigationCommands(): QuickBarItem[] {
+    const panelItems = this._generateNavigationPanelCommands();
+    const sectionItems = this._generateNavigationConfigSectionCommands();
+
+    return this._withNavigationActions([...panelItems, ...sectionItems]);
+  }
+
+  private _generateNavigationPanelCommands(): Omit<
+    QuickBarNavigationItem,
+    "action"
+  >[] {
+    return Object.keys(this.hass.panels).map((panelKey) => {
+      const panel = this.hass.panels[panelKey];
+
+      return {
+        text: this._getPanelText(panel),
+        icon: "hass:robot",
+        path: `/${panel.url_path}`,
+      };
+    });
+  }
+
+  private _generateNavigationConfigSectionCommands(): Partial<
+    QuickBarNavigationItem
+  >[] {
+    const items: NavigationInfo[] = [];
+
+    for (const sectionKey of Object.keys(configSections)) {
+      for (const page of configSections[sectionKey]) {
+        if (canShowPage(this.hass, page)) {
+          if (page.component) {
+            const info = this._getNavigationInfoFromConfig(
+              sectionKey,
+              page.component
```

Comment:
Why not pass `page`? Now we need to search the page again in `_getNavigationInfoFromConfig`?

Reply:
Good point :P

Reply:
done